### ISSUE TITLE
Stillbirth/miscarriage in different subontologies

### ISSUE THREAD

**Issue by pdl** [obophenotype/human-phenotype-ontology#6842]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0005268 Spontaneous abortion (miscarriage)

**Suggested revision and reasons**
In order to be able to assign HPO terms for patients when they have died, we have looked at this term, but it does not sit within the Aging/Mortality tree https://hpo.jax.org/app/browse/term/HP:0011420 which does not have a term for a foetus miscarried before 20 weeks, which is when stillbirth https://hpo.jax.org/app/browse/term/HP:0003826 is defined to start.

It's not totally clear to me from the description whether this term can used on the parent or the child or both, but it looks as if it is only for the parent as it has a sub-term which is 'Recurrent spontaneous abortion' HP:0200067, which can only be applied to the parent; if so, another term might be needed instead.

Note also that I understand from clinicians that the cutoff for when the event is classed as a miscarriage varies between countries - US being 20w and UK being 24 ( https://www.nhs.uk/conditions/miscarriage/ ), so it might be preferable to have a term for pregnancy loss in general with stillbirth and miscarriage being subterms.

**Comment by pnrobinson** [comment]
As currently implemented in HPO, the term [Spontaneous abortion HP:0005268](https://hpo.jax.org/app/browse/term/HP:0005268) refers to the mother, and the term  [Stillbirth HP:0003826](https://hpo.jax.org/app/browse/term/HP:0005268) refers to the fetus.
We are currently in the process of hosting workshops on the prenatal phenotype with the intention of improving the HPO's representation of these concepts. I will add the prenatal label to that and we will collect suggestions/feedback. You would be welcome to join these discussions -- if so, please send Peter Robinson an email (the email can be found on the HPO page).

**Comment by pnrobinson** [comment]
@pdl sorry for the delayed response. Looking at this, I think that we need to update the definitions for recurrent spontaneous abortion (to refer to the mother). The terminology for stillbirth/miscarriage is not uniform across countries, but I agree that it would be good to work on terms to cover prenatal demise more comprehensively. Spontaneous abortion seems to partially overlap with stillbirth, and we will need to decide how to manage the concepts. If you have literature or suggestions, for instance, existing classifications, that might be of use, please post them.

**Comment by pdl** [comment]
Thanks @pnrobinson - I don't have anything to hand, but I know a few people who might and will ask.

**Comment by pnrobinson** [comment]
I am adding a term 
Fetal death (Note we include UK spelling as a synonym type with the label "UK spelling", i.e., Foetal death etc)
--Stillbirth
--Miscarriage

The onset and death terms in the HPO are not meant to capture the precise dates of individual cases. For this we recommend the new [GA4GH Phenopacket](https://www.medrxiv.org/content/10.1101/2021.11.27.21266944v1). The HPO terms are intended to provide the general age range in ways that can be used by algorithms. So I would suggest for now that we stick with the above three terms. A possible reason for extending the hierarchy in the HPO would be if the information of say 20 vs 22 vs 24 weeks is useful for the differential diagnosis (I do not think this is the case). Otherwise, this information can be better coded using a phenopacket for individual patients. I will add synonyms with appropriate modifications.

I have left "Recurrent spontaneous abortions" as a maternal phenotype.

Thank you for these suggestions! They will be available at the next HPO release next week.

**Comment by PaulNSchofield** [comment]
Just late to this... I do think that first trimester spontaneous pregnancy loss, second trimester spontaneous loss etc. would be useful as they tend to differentiate between placental and cardiovascular origins. this could also be articulated as first trimester fetal death etc.
